prime-router/settings/organizations.yml

```diff
@@ -2125,9 +2125,16 @@
         filePath: ./upload
         credentialName: DEFAULT-SFTP
 
-
-
-
+- name: historytest
```

Comment:
there's a bunch of orgs with `ignore-` in front of them should this follow that pattern?

Reply:
Not sure I understand the question.
This has to be an organization distinct from the 'ignore' organization, so that we don't get other smoke test results mixed in with history test results.

Reply:
got it